packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { IButtonComponent, IButtonViewProps } from './Button.types';
 import { Text } from '../../Text';
-import { HorizontalStack } from '../../Stack';
+import { HorizontalStack } from 'office-ui-fabric-react/lib/Stack';
```

Comment:
This should just be absorbed into line 5 (don't need to import from lib anymore)